app/assets/javascripts/pageflow/editor/initializers/setup_file_types.js

```diff
@@ -21,7 +32,44 @@ pageflow.app.addInitializer(function(options) {
 
   pageflow.editor.fileTypes.register('text_track_files', {
     model: pageflow.TextTrackFile,
-    matchUpload: /vtt$/
+    matchUpload: /vtt$/,
+    skipUploadConfirmation: true,
+    configurationEditorInputs: [
+      {
+        name: 'label',
+        inputView: pageflow.TextInputView
+      },
+      {
+        name: 'kind',
+        inputView: pageflow.SelectInputView,
+        inputViewOptions: {
+          values: pageflow.config.availableTextTrackKinds,
+          texts: _.map(pageflow.config.availableTextTrackKinds, function(kind) {
+            var translation = 'pageflow.config.text_track_kind.' + kind;
+            return I18n.t(translation);
+          })
```

Comment:
We can use the [translationKeyPrefix](https://doclets.io/codevise/pageflow/master#dl-module-pageflow/ui-pageflow-SelectInputView) option here.